modules/apps/transfer/keeper/msg_server_test.go

```diff
@@ -131,14 +139,26 @@ func (suite *KeeperTestSuite) TestMsgTransfer() {
 				suite.Require().NoError(err)
 				suite.Require().NotNil(res)
 				suite.Require().NotEqual(res.Sequence, uint64(0))
-
-				events := ctx.EventManager().Events()
-				suite.assertTransferEvents(events, coin, "memo")
 			} else {
 				suite.Require().Error(err)
 				suite.Require().Nil(res)
+			}
 
-				events := ctx.EventManager().Events()
+			// Verify events
+			events := ctx.EventManager().Events()
+			expEvents := ibctesting.EventsMap{
+				"ibc_transfer": {
+					"sender":   suite.chainA.SenderAccount.GetAddress().String(),
+					"receiver": suite.chainB.SenderAccount.GetAddress().String(),
+					"amount":   coin.Amount.String(),
+					"denom":    coin.Denom,
+					"memo":     "memo",
+				},
+			}
```

Comment:
I think we can use the event attributes here for the keys.

https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/types/events.go

Reply:
See [discussion](https://github.com/cosmos/ibc-go/issues/2823#issuecomment-1326286697)